tests/AbstractMethodsClassTest.php

```diff
@@ -53,33 +53,33 @@ public function check_is_abstract_method_test()
         $class = ClassMethods::read($tokens);
         $methods = $class['methods'];
         // Checks all the methods are abstract
-        $this->assertEquals(true, $methods[0]['is_abstract']);
-        $this->assertEquals(true, $methods[1]['is_abstract']);
-        $this->assertEquals(true, $methods[2]['is_abstract']);
-        $this->assertEquals(true, $methods[3]['is_abstract']);
-        $this->assertEquals(true, $methods[4]['is_abstract']);
-        $this->assertEquals(true, $methods[5]['is_abstract']);
-        $this->assertEquals(true, $methods[6]['is_abstract']);
-        $this->assertEquals(true, $methods[7]['is_abstract']);
-        $this->assertEquals(true, $methods[8]['is_abstract']);
-        $this->assertEquals(true, $methods[9]['is_abstract']);
-        $this->assertEquals(true, $methods[10]['is_abstract']);
-        $this->assertEquals(true, $methods[11]['is_abstract']);
```

Comment:
Do not include these changes in this PR.

Reply:
OK, I deleted this commit